main.css

```diff
@@ -0,0 +1,334 @@
+    display: flex;
+    gap: 4px;
+}
+
+
+
+
+
+/**
+=================================================
+================== header =================
+==================================================
+**/
+
+.header {
+    display: flex;
+    justify-content: center;
+    position: relative;
+    z-index: 1;
+    background-color: #FFF5F0;
+    height: 90px;
+}
+.header__links {
+    display: flex;
+    width: 1100px;
+    gap: 21px;
+    list-style: none;
+    justify-content: center;
+    align-items: center;
+}
+.logo {
+    font-size: 24px;
+    letter-spacing: 0.1px;
+    font-weight: bold;
+    color: #252B42;
+}
+.header__item--big-gap {
+    margin-right: 67px;
+}
+.header__item--medium-gap {
+    margin-right: 45px;
+}
+.header__item--auto-gap {
+    margin-right: auto;
+}
+
+
+
+
+/**
+=================================================
+================== main section =================
+==================================================
+**/
+.banner {
+    height: 686px;
+    background-color: #FFF5F0;
+    display: flex;
+    align-items: center;
+}
+.banner__card {
+    margin-left: 132px;
+}
+.tag {
+    color: #1BA0AB;
+    font-size: 16px;
+    font-weight: bold;
+    
+}
+.banner__title {
+    max-width: 542px;
+    margin-top: 30px;
+    margin-bottom: 30px;
+}
+.banner__button-container {
+    display: flex;
+    gap: 10px;
+    margin-top: 30px;
+}
+
+/**
+=================================================
+================== experts section =================
+==================================================
+**/
+
+.experts {
+    display: flex;
+    padding-top: 80px;
+    padding-bottom: 80px;
+    gap: 144px;
+}
+.experts__img-wrapper, .experts__body {
+    flex: 1;
+}
+.experts__body {
+    align-self: center;
+    padding-top: 42px;
+    position: relative;
+}
+.experts__body-border {
+    width: 94px;
+    border-bottom: 7px solid #E74040;
+    position: absolute;
+    top: 0;
+    left: 0;
+}
+.experts__title {
+    margin-bottom: 35px;
+}
+.experts__text {
+    margin-bottom: 35px;
+}
+.experts__img-wrapper {
+    height: 550px;
+    width: 520px;
+}
+
+/**
+=================================================
+================== team section =================
+==================================================
+**/
+
+.team {
+    margin-bottom: 112px;
+}
+.team__title {
+    padding-top: 10px;
+    padding-bottom: 10px;
+}
+.team__text {
+    max-width: 470px;
+    margin-bottom: 112px;
+}
+
+/**
+=================================================
+================== newsletter section =================
+==================================================
+**/
+.newsletter {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: #FFF5F0;
+    padding-top: 160px;
+    padding-bottom: 160px;
+    text-align: center;
+}
+.newsletter__text {
+    max-width: 470px;
+    margin-bottom: 80px;
+}
+.newsletter__title {
+    margin-top: 10px;
+    margin-bottom: 10px;
+}
+.input {
```

Comment:
Introducing this block here would not be a good idea, it's better to have related blocks and modifiers, and elements in the same place. you can place it done with that placeholder selector